com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/RobustContrastAdaptiveSharpen.compute

```diff
@@ -0,0 +1,63 @@
+#pragma kernel KMain
+#pragma kernel KInitialize
+
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+#include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl"
+
+#define A_GPU 1
+#define A_HLSL 1
+#define FSR_RCAS_F 1
+
+#include "Packages/com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/ffx_a.hlsl"
+
+// two elements:
+// [0] = const0
+// [1] = const1
+// ComputeBuffer is allocated with stride sizeof(int)*4, 2 elements
```

Comment:
Looks like it is actually a single uint4 and not 2 elements

Reply:
yep. Copy pasted from CAS. Fixing this and all comments.